### PR TITLE
Create zh_TW.json

### PR DESCRIPTION
**Pull request by assanges**
Translate '/locales/fr.json' in 'zh_TW' (Taiwanese Mandarin)

### CODE REVIEW

File: locales/zh_TW.json [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+      "error": "應為空值或至少10個字元長"
+    },
+    "submit": "發佈",
+    "cancel": "取消"
+  },
+  "details": {
+    "signal_warning": "您正在回報某項設施的資訊，若您離開將不會保存任何更動。仍要繼續嗎？",
+    "signal_state": {
+      "open": "塡寫資訊",
+      "open_adapted": "塡寫資訊,
```
Comment (frodrigo) [inline locales/zh_TW.json:106]:
```suggestion
      "open_adapted": "塡寫資訊",
```
```suggestion
      "open_adapted": "塡寫資訊,
```

File: locales/zh_TW.json [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+    "motorcycle": "機車",
+    "bicycle_tube": "自行車輪胎販賣機",
+    "tyres": "輪胎"
+  },
+  "qa": {
+    "title": "圖資品質保證(QA)"
+  },
+  "bycity": {
+    "title": "商店列表",
+    "loading": "載入中,
```
Comment (frodrigo) [inline locales/zh_TW.json:360]:
```suggestion
    "loading": "載入中",
```

**Comment by frodrigo** [comment]
I fixed first the json, but it is ok now. Thank you. Merged.

**Comment by assanges** [comment]
Merci bien 🎉

**Pull request merged**